src/index.js

```diff
@@ -44,7 +45,8 @@ class ReactTooltip extends Component {
     afterHide: PropTypes.func,
     disable: PropTypes.bool,
     scrollHide: PropTypes.bool,
-    resizeHide: PropTypes.bool
+    resizeHide: PropTypes.bool,
+    style: PropTypes.object
```

Comment:
In IE11, if the parent element for the component for the tooltip has a transform,translate(-50%, -50%), then the tooltip will not render in the correct location. By enabling a style prop to be passed down to the tooltip, a developer then can resolve the issue by passing a style prop with {position: 'absolute'} for the tooltip.